src/lib/shopping-analytics.js

```diff
@@ -64,7 +64,7 @@ export const shoppingAnalyticsSetup = (config? : Config = {}) => {
   }
   
   debugLogger.log('[shopping-analytics:shoppingAnalyticsSetup] Initialized shopping analytics with the following configuration', config);
-
+  // TODO: promisify setupUserDetails, setupContainer
```

Comment:
Let's remove todo and create ticket

Reply:
Comment removed. I'm not sure is it worth separate ticket. Benefits are marginal.